content/tutorial/tutorial.md

```diff
@@ -1070,28 +1078,29 @@ to
 <li>Alexa: 5 tasks left</li>
 ```
 
-In addition to the updated counts, a human reading this would probably say that we swapped Alexa and Ben's ordering and inserted Claudia between Alexa and Ben. However, React is a computer program and does not know what we intended. Because React cannot know our intentions, we need to specify a *key* property for each list item to differentiate each list item from its siblings. One option would be to use the strings `alexa`, `ben`, `claudia`. If we were displaying data from a database, Alexa, Ben, and Claudia's database IDs could be used as keys.
+अपडेटेड काउंट्स के अलावा, यह पढ़ने वाला एक मानव शायद यह कहेगा कि हमने एलेक्सा और बेन के ऑर्डर को स्वैप किया और एलेक्सा और बेन के बीच क्लाउडिया को डाला। हालाँकि, React एक कंप्यूटर प्रोग्राम है और यह नहीं जानता कि हमने क्या इरादा किया था। क्योंकि React हमारे इरादों को नहीं जान सकती, इसलिए हमें प्रत्येक सूची आइटम के लिए उसके सिब्लिंग्स से अलग करने के लिए प्रत्येक सूची आइटम के लिए एक *कुंजी* गुण निर्दिष्ट करने की आवश्यकता है। 
```

Comment:
should we translate एलेक्सा, ben and Claudia? If yes, do so in code above too?

Reply:
और यह नहीं जानता कि हमने क्या इरादा किया था-> humaara kya iraada tha

Reply:
क्योंकि React हमारे इरादों को नहीं जान सकती, इसलिए हमें प्रत्येक सूची आइटम के लिए उसके सिब्लिंग्स से अलग करने के लिए प्रत्येक सूची आइटम के लिए एक *कुंजी* गुण निर्दिष्ट करने की आवश्यकता है।  -> revisit

Reply:
> should we translate एलेक्सा, ben and Claudia? If yes, do so in code above too?

I think we should keep it as it is

Reply:
> क्योंकि React हमारे इरादों को नहीं जान सकती, इसलिए हमें प्रत्येक सूची आइटम के लिए उसके सिब्लिंग्स से अलग करने के लिए प्रत्येक सूची आइटम के लिए एक _कुंजी_ गुण निर्दिष्ट करने की आवश्यकता है। -> revisit

```
React हमारे इरादों को नहीं जान सकता की हम कोड से क्या करना चाहते हैं इसलिए हमें लिस्ट आइटम और उसके सिब्लिंग्स को अलग करने के लिए एक *key* प्रॉपर्टी हर लिस्ट आइटम पर लगाते हैं। 
```
?

Reply:
cool